include_core/omrport.h

```diff
@@ -1775,6 +1775,20 @@ typedef struct OMRProcessorDesc {
 #define OMR_FEATURE_X86_XSAVE_AVX512 64 + 2 /* OS Supports AVX-512 (zmm, opmask) state */
 #define OMR_FEATURE_X86_XSAVE_APX    64 + 3 /* OS Supports APX (r16-r31) state */
 
+/* FLAGS FOR AVX 10.X SUPPORT.
+ * These flags are to be set after checking for AVX10 and XCR0 register flags.
+ */
+#define OMR_FEATURE_X86_AVX10_1      64 + 4  /* AVX 10.1 */
+#define OMR_FEATURE_X86_AVX10_2      64 + 5  /* AVX 10.2 */
```

Comment:
formatting should be kept consistent with surrounding code; extra space before the comment should be removed
```suggestion
#define OMR_FEATURE_X86_AVX10_1      64 + 4 /* AVX 10.1 */
#define OMR_FEATURE_X86_AVX10_2      64 + 5 /* AVX 10.2 */
```